internal/logger/logger.go

```diff
@@ -42,6 +42,14 @@ var (
 	defaultLogger        *slog.Logger
 )
 
+// SLogger exports the slog logger.
+// This allows one to use the full breadth of features provided by slog.
+//
+// TODO: export defaultLogger instead of using this function.
+func SLogger() *slog.Logger {
```

Comment:
Could we not use logger.Info() ?

Reply:
[logger.Info()](https://github.com/GoogleCloudPlatform/gcsfuse/blob/340fa01712e8e8231d4dcc71e7b5bcf12fda96ba/internal/logger/logger.go#L145) takes the arguments and converts them to a single string. So, the structure is lost. Let me know if I am missing something.

Reply:
Changed the implementation of `logger.Info(message string, args ...any)` to be a dumb wrapper around `slog.Info(message, args ...any)`